examples/delegator-macro/src/lib.rs

```diff
@@ -1,15 +1,16 @@
-use near_abi_client_macros::near_abi_client;
 use workspaces::network::DevAccountDeployer;
 
-near_abi_client! { type AbiClient for "src/adder.json" }
+mod adder {
+    near_abi_client::generate!(Client for "src/adder.json");
+}
```

Comment:
I put this in a nested module to avoid the types generated from this invocation potentially clashing with types already defined in this file.